### PR TITLE
Feature/analysis pages

### PR DESCRIPTION
**Pull request by simvadnbu**
This adds 2 analysis pages. 

1. `admin/analysis/worked-hours`: You get the worked hours of a student within a period, you can also view the tours a student has done.
2. `admin/analysis/student-on-tour?studentOnTour=...`: You get an overview of the buildings a student has done on a specific tour + time analysis of each building.

  
image

When a student is 25% faster/slower than the expected duration the bar will be full (to the left or right), you can hover over the progress bar to get the specific times.

How can you test this?

I added a datadump for student `stephan test` on this date `13/5` for tour `Centrum (Gent)`, you can just view his analysis.

### CODE REVIEW

**Review by emneirin — APPROVED** [review]
Looks really good!
Maybe we could also delete the index.tsx file in ananlysis?

**Comment by simvadnbu** [comment]
> Looks really good! Maybe we could also delete the index.tsx file in ananlysis?

That's true, I removed it.

**Review by n00bS-oWn-m3 — APPROVED** [review]
Looks great! 
Little remark: personally, it doesn't make much sense to me that tours from the past that weren't ever started are annotaded with `(gepland)`. Maybe we can either not show these tours or provide some other indication

**Comment by TiboStr** [comment]
I recreated database with new dump en rebuilt Docker.

I see that the frontend wishes to fetch `http://localhost:2002/undefined?start_date=2023-04-16&end_date=2023-05-16`


image

**Comment by TiboStr** [comment]
> I recreated database with new dump en rebuilt Docker.
> 
> I see that the frontend wishes to fetch `http://localhost:2002/undefined?start_date=2023-04-16&end_date=2023-05-16`
> 
> image

Problem was orphan containers. My bad.

**Review by TiboStr — APPROVED** [review]

**Comment by GashinRS** [comment]
Is there a way to see the total average of a tour? So basically an overview of the average time it takes to finish a tour, calculated from the average of every time this tour was completed. I would expect this to work in a similar way to how a works for a single building, so with the same bar with the planned time and the actual average.

Also, while these progress bars look quite fancy, I think it's better to also add a numeric value somewhere without having to hover every single value, like `2 min 30s` or `5 min 20s` (color coded) because it can be hard to see just how fast or slow a student is solely based on these progress bars. The following bar can be quite misleading for example because the student here is only about one minute slower than the expected time and it already fills almost half of the right bar.
image

Maybe you could add this in the middle on top of every bar? How exactly this looks is not a priority right now, but I do think it would be a nice to have and it should (hopefully) not be too much work.

So basically it would look something like this, while you can still keep the hover over values to see the total amount of time a building took (the values in my edit are not accurate):
image

A sum of of all those numbers would be nice too, since being particularly fast or slow at a single building does not tell the entire story, the total time for the tour matters as well. For this, you could either add another bar like the ones you already have or maybe just add the number somewhere on top next to the tour name or somewhere at the bottom like a small summary

**Review by jonathancasters — APPROVED** [review]

**Review by jonathancasters — CHANGES_REQUESTED** [review]
There are some features that are missing for it to be perfect: 
1. Filters on region
2. Sorting on alphabet / highest to lowest work hours / ...

**Review by jonathancasters — APPROVED** [review]
